### ISSUE TITLE
Failed resolution of Lio/opentelemetry/api/trace/Span

### ISSUE THREAD

**Issue by slowcar** [AzureAD/microsoft-authentication-library-for-android#1715]
I have an error in the authentication process of an account, it occurs with msal version 4.0.3 (and also 4.0.4), but not with 4.0.2

We have a two step authentication process as we utilize both EWS and Graph API.
The authentication with the ews scope is successful, afterwards we try to get a second access token for the Graph API, which fails.

The account in question is missing administrator consent to a new graph scope, User.Read.All, that we added recently.
If i use version 4.0.2 the oauth webpage shows the missing administrator consent (correct), but if i change to 4.0.3 or 4.0.4 i get a MsalClientException with mErrorCode unknown_error and detailMessage "failed resolution of Lio/opentelemetry/api/trace/Span"

see [screenshot](https://i.imgur.com/7RXtsTZ.png)

The authentication process is successful for other accounts where the new graph permission/scope was already granted.
It is also successful if i remove the new scope.

To summarize: Adding a new scope to graph on an account where it needs administrator consent causes an error in msal since 4.0.3

**Comment by shahzaibj** [comment]
@slowcar What is the minimum SDK version for your application? Also do you have minification enabled or disabled for your app?

**Comment by slowcar** [comment]
> @slowcar What is the minimum SDK version for your application? Also do you have minification enabled or disabled for your app?

Our current minSDK is 25, targetSDK 33. The issue occurrs on the debug build, no minificiation enabled there.

**Comment by shahzaibj** [comment]
> > @slowcar What is the minimum SDK version for your application? Also do you have minification enabled or disabled for your app?
> 
> Our current minSDK is 25, targetSDK 33. The issue occurrs on the debug build, no minificiation enabled there.

@slowcar We've checked in the fix here: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/1894

This will get released with next release of MSAL and that should be available within 1-2 weeks.
